CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ option(FETCH_ENABLE_COVERAGE            "Enable coverage generation"
 option(FETCH_ENABLE_METRICS             "Enable the collection of metrics"      OFF)
 option(FETCH_DISABLE_COLOUR_LOG         "Disable the colour logging"            OFF)
 option(FETCH_STATIC_LINK                "Enable static linking"                 OFF)
-option(FETCH_ENABLE_LTO                 "Enable link-time optimisation"         ON)
+option(FETCH_ENABLE_LTO                 "Enable link-time optimisation"         OFF)
```

Comment:
?

Reply:
If this is for the docker build this can be reverted because I have fixed this issue in the latest version of the `infra-constellation` project

Reply:
yes, this was not intended to slip in. thanks for fixing it, I will revert this